extensions/objects/protractor.js

```diff
@@ -179,6 +179,19 @@ var ParameterNameEditor = function(elem) {
   };
 };
 
+var PositiveIntEditor = function(elem) {
+  return {
+    setValue: async function(value) {
+      await elem.element(by.tagName('input')).clear();
+      await elem.element(by.tagName('input')).sendKeys(value);
+    },
+    expectValueToBe: async function(expectedValue) {
+      var value = await elem.element(by.tagName('input')).getAttribute('value');
```

Comment:
Can you wait for the element to be visible before getting its attribute?

Reply:
Done.